knora-ontologies/knora-base.ttl

```diff
@@ -2703,11 +2920,99 @@ rdfs:label rdfs:subPropertyOf :objectCannotBeMarkedAsDeleted .
                                                "Representation (Text)"@en ,
                                                "Répresentation (Texte)"@fr ,
                                                "Rappresentazione (testo)"@it ;
-                    
+
                     rdfs:comment "A resource containing text files"@en .
 
 
 
+###  http://www.knora.org/ontology/knora-base#Person
+
+:Person rdf:type owl:Class ;
+
+        rdfs:subClassOf :Resource, foaf:Person,
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasFamilyName ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasGivenName ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasIAFIdentifier ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasBirthDate ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ],
+                        [ rdf:type owl:Restriction ;
+                          owl:onProperty :hasDeathDate ;
+                          owl:maxCardinality "1"^^xsd:nonNegativeInteger
+                        ];
+
+        rdfs:label "Person"@de ,
+                   "Person"@en ,
+                   "Personne"@fr ,
+                   "Persona"@it ;
+
+        rdfs:comment "A resource representing a person"@en .
+
+
+###  http://www.knora.org/ontology/knora-base#Publication
+
+:Publication rdf:type owl:Class ;
```

Comment:
I don't think I would completely agree with those new additions. 

To me, they are really restrictive and project-specific. What if you want to use a `Book` class, as a subclass of `Publication`, but in this ontology authors are nodes from a list value? What if `Author` is not a single `Person` but a collective or an organisation?  What if `Author` is anonym and consequently wouldn't fullfill the properties required for a `Person`? Then, such project will have to build its own ontology, without possibility to link their data to the broader general ontology.

And don't you think those properties are not so "basic" anymore and not related to "knora-base" grammar but somewhat to a vocabulary that might be common to various projects ? If so, shouldn't it be in another named graph and couldn't it allow a little bit more flexibility (I am just asking)? Imagine doing the same in a more painting-specific perspective, or define what should be a `Location`, or an `Archeological site` in knora-base.ttl... I don't think it would make sense but it might in a separate named graph.

I mean, the old Salsah was all about allowing project to define their own data model. I don't think binding together data from various projects should result in such strong prescriptions. But maybe I am wrong... (and maybe there is no way to do otherwise at the moment)?